__tests__/app.test.js

```diff
@@ -232,4 +240,72 @@ describe("GET /api/articles", () => {
         });
     });
   });
+
+  describe("PATCH /api/articles/:article_id", () => {
+    it(" PATCH /updates vote property by article_id", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 6 })
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { article },
+          } = response;
+          expect(article).toMatchObject({
+            votes: 6,
+          });
+        });
+    });
+    it(" PATCH /updates vote property by article_id for minus numbers", () => {
+      return request(app)
+        .patch("/api/articles/4")
+        .send({ inc_votes: -15 })
+        .expect(200)
+        .then((response) => {
+          const {
+            body: { article },
+          } = response;
+          expect(article).toMatchObject({
+            votes: -15,
+          });
+        });
+    });
+
+    it("PATCH / returns 404 msg for valid id but non-existent article", () => {
+      return request(app)
+        .patch("/api/articles/67985986")
+        .send({ inc_votes: 1 })
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Not found");
+        });
+    });
+    it("PATCH/ returns 400 msg for invalid id", () => {
+      return request(app)
+        .patch("/api/articles/not-an-id")
+        .send({ inc_votes: 1 })
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Invalid request");
+        });
+    });
+    it("PATCH / returns 400 error if votes is empty", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({})
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
```

Comment:
This could be refactored into an optional status 200, if inc_votes is missing, should not update article and return original single article object